package.json

```diff
@@ -10,26 +10,16 @@
     "node": ">=18.0.0"
   },
   "scripts": {
-    "prepare": "husky",
     "publish-modules": "npx lerna run build && npx lerna publish",
     "build": "npx lerna run build",
-    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --fix && prettier --write \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints --fix",
+    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx && prettier --write \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints",
+    "lint:js:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix && prettier --write \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints --fix",
     "lint:clang": "./scripts/run-clang-format.sh",
     "lint:swift": "./scripts/run-swiftlint.sh",
```

Comment:
there's a "fix" flag used in swiftlint in `run-swiftlint.sh` that should probably also be removed or made optional, I think the same goes for `run-clang-format.sh`